ci-operator/step-registry/openstack-k8s-operators/sno/scale-down/openstack-k8s-operators-sno-scale-down-commands.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -ex
+
+oc patch etcd cluster -p='{"spec": {"unsupportedConfigOverrides": {"    useUnsupportedUnsafeNonHANonProductionUnstableEtcd": true}}}' --type=merge
+oc patch authentications.operator.openshift.io cluster -p='{"spec": {"unsupportedConfigOverrides": {"useUnsupportedUnsafeNonHANonProductionUnstableOAuthServer": true}}}' --type=merge
+
+oc scale --replicas=1 ingresscontroller/default -n openshift-ingress-operator
+oc scale --replicas=1 deployment.apps/console -n openshift-console
+oc scale --replicas=1 deployment.apps/downloads -n openshift-console
+oc scale --replicas=1 deployment.apps/oauth-openshift -n openshift-authentication
+oc scale --replicas=1 deployment.apps/packageserver -n openshift-operator-lifecycle-manager
+
+oc scale --replicas=0 deployment.apps/prometheus-{operator,adapter} -n openshift-monitoring
+oc scale --replicas=0 deployment.apps/thanos-querier -n openshift-monitoring
+oc scale --replicas=0 deployment.apps/telemeter-client -n openshift-monitoring
+oc scale --replicas=0 deployment.apps/cluster-monitoring-operator -n openshift-monitoring
+oc scale --replicas=0 deployment.apps/{grafana,openshift-state-metrics,kube-state-metrics} -n openshift-monitoring
+
+oc delete project openshift-monitoring
```

Comment:
I'm wondering if in this context having something like:

```
oc delete all --all -n openshift-monitoring
oc delete project openshift-monitoring
```
is enough, or if just an `oc delete project` triggers the `delete` of all the resources defined on that namespace.
If the suggestion above works, you can just remove L14-L18 and rely on L20.

Reply:
In my tests I see that `oc delete project <namespace>` does the trick: all the resources are `Terminated` first and properly `deleted`! I guess L20 is enough to get rid of the `monitoring` namespace.

Reply:
I will try that and update this step, thanks Francesco.